packages/boltzgen/boltzgen.0.9/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Generate test using boltzman sampling"
+description:
+  "BoltzGen is a tool to generate tests. Given a function signature it generates a random set of call of this function on generated random input. Inputs are generated using a Boltzman sampler for unlabeled structure."
+maintainer: ["barbot@lacl.fr"]
+authors: ["Benoît Barbot"]
+license: "GPLv3"
+homepage: "https://git.lacl.fr/barbot/boltzgen"
+bug-reports: "https://git.lacl.fr/barbot/boltzgen/-/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "ocaml" {>= "4.08.1"}
+  "ocaml-compiler-libs"
+  "cmdliner"
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+url {
+  src:
+    "https://git.lacl.fr/barbot/boltzgen/-/archive/master/boltzgen-master.tar.gz"
```

Comment:
I don't think this is the archive you want to release. Unless I'm mistaken, this archive will change overtime and the installation of this package will fail because the checksum is not going to be correct.

Could you use the archive of the tag instead?